Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -36,16 +36,79 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
                 (await multipleAsync.ReadAsync<Entities.ContactPoliciesSettings>()).FirstOrDefault();
 
             if (contactPoliciesSettings == null)
+            {
                 return null;
+            }
+
             if (contactPoliciesSettings.IdUser == null)
+            {
                 return contactPoliciesSettings;
+            }
 
             contactPoliciesSettings.ExcludedSubscribersLists =
                 (await multipleAsync.ReadAsync<ExcludedSubscribersLists>()).ToList();
 
             return contactPoliciesSettings;
         }
 
+        public async Task<Entities.ContactPoliciesSettings> GetBasicContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select cp.IdUser, cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
+                from [User] u
+                left join [UserShippingLimit] cp on u.IdUser = cp.IdUser and cp.Enabled = 1
+                where u.Email = @Email;";
+
+            var queryParams = new { Email = accountName };
+            var contactPoliciesSettings = await connection.QueryFirstOrDefaultAsync<Entities.ContactPoliciesSettings>(query, queryParams);
+
+            if (contactPoliciesSettings == null)
+            {
+                return null;
+            }
+
+            if (contactPoliciesSettings.IdUser == null)
+            {
+                return contactPoliciesSettings;
+            }
+
+            return contactPoliciesSettings;
```

Comment:
Have we differences with the following code?
```suggestion
            return await connection.QueryFirstOrDefaultAsync<Entities.ContactPoliciesSettings>(query, queryParams);
```

Reply:
i dontk think we have a difference in there, i can just return directly, ok, noted!